.github/workflows/codecov.yml

```diff
@@ -0,0 +1,24 @@
+name: Coverage
+
+on:
+  push:
+    branches: ['main']
+
+jobs:
+  coverage:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          toolchain: stable
+          override: true
+      - uses: actions-rs/tarpaulin@v0.1
+      - uses: codecov/codecov-action@v1.0.2
+        #with:
```

Comment:
@djc IIUC the token is only needed for private repos?

Reply:
Yup, should not longer be needed. But also see https://github.com/quinn-rs/quinn/pull/1505.

Reply:
Thanks! Updated to the new codecov.